migrations/20180402170153_make-varchar-uniform.js

```diff
@@ -0,0 +1,156 @@
+/* eslint-disable */
+exports.up = async knex => {
+  await knex.schema.alterTable('articles', table => {
+    table
+      .string('slug')
+      .notNullable()
+      .alter();
+    table
+      .string('title')
+      .notNullable()
+      .alter();
+  });
+  await knex.schema.alterTable('categories', table => {
+    table
+      .string('slug')
+      .notNullable()
+      .alter();
+    table
+      .string('name')
+      .notNullable()
+      .alter();
+  });
+  await knex.schema.alterTable('info_pages', table => {
+    table
+      .string('slug')
+      .notNullable()
+      .alter();
+    table
+      .string('title')
+      .notNullable()
+      .alter();
+  });
+  await knex.schema.alterTable('issues', table => {
+    table
+      .string('name')
+      .notNullable()
+      .alter();
+  });
+  await knex.schema.alterTable('semesters', table => {
+    table
+      .string('name')
+      .notNullable()
+      .alter();
+  });
+  await knex.schema.alterTable('staff', table => {
+    table
+      .string('slug')
+      .notNullable()
+      .alter();
+    table
+      .string('name')
+      .notNullable()
+      .alter();
+    table.string('job_title').alter();
+    table.string('biography', 1000).alter();
+    table.text('image_url').alter();
```

Comment:
My purpose was to actually make this a `string` aka varchar, since when I was creating this I at first thought we may be storing the whole image, but we're only storing URLs at the moment